### ISSUE TITLE
How to disable mustache?

### ISSUE THREAD

**Issue by vowar** [mashpie/i18n-node#436]
I am working with a framework that uses mustache syntax to handle validation error message, but because of i18n mustache implementation, I can't use them both. Can I disable mustache support in the i18n-node module somehow?

**Comment by mashpie** [comment]
Unfortunately there is now feature flag to disable mustache rendering by now. No big deal to implement, though. I am also thinking about providing custom delimiters https://www.npmjs.com/package/mustache#custom-delimiters. No eta yet bot on roadmap to 0.11.0

**Comment by mashpie** [comment]
There is a new option available since 0.11.0:

```js
i18n.configure({
    // [...]

    // use mustache with customTags (https://www.npmjs.com/package/mustache#custom-delimiters) or disable mustache entirely 
    mustacheConfig: {
      tags: [ '{{', '}}' ],
      disable: false
    },
});
```

to just disable mustache:

```js
i18n.configure({
    // [...]
    mustacheConfig: {
      disable: true
    },
});
```
